README.md

```diff
@@ -186,6 +186,21 @@ inst.file.Write(writer, 0, new List<AssetsReplacer>() { repl }, 0);
 
 Once you write changes to a file, you will need to reopen the file to see the changes.
 
+#### Create Blank Assets file
+
+To create blank assets file, get the bytes with `AssetHelper.CreateBlankAssets(string engineVersion)`, and call `File.WriteAllBytes(string path, byte[] bytes)`:
+
+```cs
+byte[] data = AssetHelper.CreateBlankAssets("2017.4.11f1");
+File.WriteAllBytes("resources.assets", data);
+```
+
+If you want to use an AssetsFile variable instead of writing bytes:
+
+```cs
+AssetsFile af = new AssetsFile(new AssetsFileReader(new MemoryStream(AssetHelper.CreateBlankAssets("2017.4.11f1"))));
+```
+
```

Comment:
I would say to condense this into one code block and maybe making the reader its own local. Also for consistency in the readme, replacing the types with var. I would also remove the ", and call `File.WriteAllBytes`" from the top since that's general c# knowledge and could be said about any of the functions in the readme that spit out byte arrays.